docs/pages/eas-update/develop-faster.mdx

```diff
@@ -1,42 +1,20 @@
-
-To ensure that we're using the modern version of the updates protocol, make sure to run `eas update:configure`, then `npx expo install expo-updates` before creating development builds. Running both of these commands should set up our local development environment to use the modern manifest protocol.
-
-## Developing with other developers
-
-When we finish a feature or bug fix, it's convenient to allow other developers to preview a published update for reviewing purposes. One problem with previewing a new feature is that we have to rebuild the project into an app and distribute it to a reviewer. That process takes time and requires bookkeeping of builds and features.
-
-Previewing features with EAS Update can help us preview new features in minutes.
-
-The process flows like this:
-
-- A developer runs `eas device:create`, then has all teammates/reviewers register their devices.
-- Once complete, they create a build suitable for internal distribution. After running `eas build:configure`, the command to generate this build would be `eas build --profile preview`.
-- Once the build is complete, all teammates/reviewers should download the internal distribution app.
-- Then, a developer makes a change to a project. For this example, let’s imagine the developer is on a branch named `typo-fix` and the change fixes a typo within the app.
-- The developer then runs `eas update --auto`. This will publish an update to a branch named `typo-fix`, which is linked by default to a channel named `typo-fix`.
-- Then, the developer can create a URL that will open the previously created internal distribution app. They can either send that link to their teammates/reviewers, or they can create a QR code that teammates/reviewers may scan. To help with this last part, developers can navigate to their public project page and see the following UI to configure the correct URL for previewing a published EAS Update:
-
-<ImageSpotlight
-  alt="Preview EAS Update QR Code configuration"
-  src="/static/images/eas-update/qr-preview.png"
-  style={{ maxHeight: '700px', width: 'auto' }}
-/>
-
-For more information on using EAS Update with development builds, see [Using expo-dev-client with EAS Update](/eas-update/expo-dev-client).
+A published update can be loaded in various environments:
+- An [expo-dev-client](/eas-update/expo-dev-client/) build with a matching runtime version. Either an [internal distribution](/build/internal-distribution/) or simulator/emulator build.
+- A build distributed via TestFlight, Google Play beta, or [internal distribution](/build/internal-distribution/) with a matching runtime version and channel pointing to a branch with the published update. With this strategy, often a EAS Update channel like "alpha" or "beta" will be used, and builds and updates will be created from development branches in source control.
```

Comment:
```suggestion
- A release build distributed via TestFlight, Google Play beta, or [internal distribution](/build/internal-distribution/) with a matching runtime version and channel pointing to a branch with the published update. With this strategy, often a EAS Update channel like "alpha" or "beta" will be used, and builds and updates will be created from development branches in source control.
```

Reply:
I made one edit to Brent's suggestion to prefer mentioning Google Play Beta before TestFlight (as per our writing [style guide's platform order](https://github.com/expo/expo/blob/main/guides/Expo%20Documentation%20Writing%20Style%20Guide.md#referencing-android-ios-and-web)).